src/TodoItem.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import './App.css';
+
+class TodoItem extends Component {
+
+    handleDelete(item_id){
+        this.props.handleDelete(item_id);
+    }
+
+    handleComplete(item_id){
+        this.props.handleComplete(item_id);
+    }
+
+    render() {
+        const{id, itemName, isCompleted} = this.props.todoItem;
+
+        return (
+            <li className={(isCompleted ? 'setLineThroughTextDeco' : 'setNormalTextDeco')}>
+                <input name="isCompleted" type="checkbox" onChange={this.handleComplete.bind(this, id)} checked={isCompleted}/>
+                {itemName}
+                <button onClick={this.handleDelete.bind(this, id)}>Delete</button>
```

Comment:
dont use this systax `this.handleDelete.bind(this, id)`. you should do it in constructor only!. this will create performance problem
https://reactjs.org/docs/faq-functions.html

> Note:
Using Function.prototype.bind in render creates a new function each time the component renders, which may have performance implications (see below).